src/vstest.console/Internal/ConsoleLogger.cs

```diff
@@ -484,11 +522,20 @@ private void TestResultHandler(object sender, TestResultEventArgs e)
                 testDisplayName = string.Format("{0} [{1}]", testDisplayName, formattedDuration);
             }
 
+            var executionId = GetExecutionId(e.Result);
+            var parentExecutionId = GetParentExecutionId(e.Result);
+
+            if (parentExecutionId != null && leafExecutionIdAndTestOutcomePairDictionary.ContainsKey(parentExecutionId))
```

Comment:
@hvinett  `GetParentExecutionId` returns non-nullable guid, removing the nullability check, as well as replacing it with `!= Guid.Empty` check passes all the tests in `ConsoleLoggerTests` what is the intent here?

Reply:
I kept parentExecutionId != null, because .containskey() throws error if key is null

Reply:
Yup. Let me quickly patch that, because I have the code open, and merge if that passes (it should, I tried it before).

Reply:
ok